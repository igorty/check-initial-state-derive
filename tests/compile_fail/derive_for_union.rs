use check_initial_state_derive::CheckInitialState;

fn main() {}

#[derive(CheckInitialState)]
union Union {
	integer: i32,
}
