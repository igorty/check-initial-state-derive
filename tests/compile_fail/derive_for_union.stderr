error: `CheckInitialState` procedural macro is allowed for structs with named fields only
 --> $DIR/derive_for_union.rs:6:1
  |
6 | / union Union {
7 | |     integer: i32,
8 | | }
  | |_^
